File: 0283-move-zeroes/0283-move-zeroes.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int currentIndex = 0;
        for (int i = 0; i < nums.size(); i++)
            if (nums[i] != 0)
                nums[currentIndex++] = nums[i];
        while (currentIndex < nums.size())
            nums[currentIndex++] = 0;
    }
};

File: 0283-move-zeroes/0283-move-zeroes.py
class Solution(object):
    def moveZeroes(self, nums):
        currentIndex = 0;
        for num in nums:
            if (num != 0):
                nums[currentIndex] = num;
                currentIndex+=1;
        for element in nums[currentIndex:]:
            nums[currentIndex] = 0;
            currentIndex+=1;
        return (nums);n